### DISCUSSION TITLE
Importing a save from the main game

### DISCUSSION THREAD

**Discussion by Orsucciu** [Melledy/Grasscutter#2139] [General]
Hello,

I guess this is probably impossible but

For me, grasscutter is mostly interesting to just play the game by eliminating the pay2win/gacha elements of the game; since i'm already kind of far, is there a way to copy where i am in the main game to my server ? I can only think of checking everything manually and copying it back here, but maaaaaaybe the client can have some local copy of the game data or something ? idk how the game is implemented

Thanks

**Comment by NotThorny** [comment]
There is nothing stored locally on your machine from the main game 'save', that's exclusively on the mhy servers. You can't import any questing or world progression, but there are several tools you can use to bring over inventory and showcase characters. You can use [InventoryKamera ](https://github.com/Andrewthe13th/Inventory_Kamera) to copy inventory data and then use [Jie's GCTools](https://github.com/jie65535/GrasscutterCommandGenerator) to import that .GOOD data.

You can try enka but it's only public showcase data and the plugin doesn't work as-is. You're better off just manually recreating your data.
